### ISSUE TITLE
WiredTiger configuration split percentage

### ISSUE THREAD

**Issue by keithbostic** [couchbaselabs/ForestDB-Benchmark#13]
The WiredTiger configuration split percentage is explicitly set to 100; this should only be done in response to specific workloads and key/value insert patterns, and is almost certainly a bad idea in a benchmark.

**Comment by greensky00** [comment]
Yes, that makes sense to me. Let me add a new option for the configuration of split percentage. Thanks.
